data/json/items/gun/5x50.json

```diff
@@ -21,7 +21,11 @@
     "range" : 10,
     "durability": 9,
     "barrel_length" : 1,
-    "burst": 10,
+    "modes": [
+      [ "DEFAULT", "semi-auto", 1 ],
+      [ "BURST", "3 rd.", 3 ],
```

Comment:
The 5x50mm stuff looks good, though I'd put the burst settings on the RM216 at 1/5/10 instead of 1/3/10.